.travis.yml

```diff
@@ -4,7 +4,7 @@ git:
 jdk:
   - oraclejdk8
 env:
+  - HADOOP_VERSION=2.6.5
   - HADOOP_VERSION=2.7.5
   - HADOOP_VERSION=2.8.3
-  - HADOOP_VERSION=2.9.0
-script: "./gradlew test -Ddyno.hadoop.bin.version=${HADOOP_VERSION}"
+script: "travis_wait ./gradlew test --stacktrace --info -Ddyno.hadoop.bin.version=${HADOOP_VERSION}"
```

Comment:
so it is possible that there is no output for 10 minutes ?

Reply:
why is 2.9.0 removed? (not as described - add in proper test files for 2.9.0)

Reply:
Looks that 2.9 is not fully compatible at the moment, so pending Issue #3. The test could potentially take > 10 minutes in some cases, but actually now that I added `--info`, it will continue producing output, so the `travis_wait` will no longer be necessary.